decode.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 func e(format string, args ...interface{}) error {
-	return fmt.Errorf("toml: "+format, args...)
+	return fmt.Errorf(format, args...)
```

Comment:
Does this result in removing the `toml` prefix from other error messages?

Reply:
Yes, I will fix this question.